verification/src/transaction_verifier.rs

```diff
@@ -513,39 +529,41 @@ const METRIC_TYPE_FLAG_MASK: u64 = 0x6000_0000_0000_0000;
 const VALUE_MASK: u64 = 0x00ff_ffff_ffff_ffff;
 const REMAIN_FLAGS_BITS: u64 = 0x1f00_0000_0000_0000;
 
-/// TODO(doc): @zhangsoledad
+/// Metric represent value
 pub enum SinceMetric {
-    /// TODO(doc): @zhangsoledad
+    /// The metric_flag is 00, `value` can be explained as a block number or a relative block number.
     BlockNumber(u64),
-    /// TODO(doc): @zhangsoledad
+    /// The metric_flag is 01, value can be explained as an absolute epoch or relative epoch.
     EpochNumberWithFraction(EpochNumberWithFraction),
-    /// TODO(doc): @zhangsoledad
+    /// The metric_flag is 10, value can be explained as a block timestamp(unix time) or a relative
     Timestamp(u64),
 }
 
-/// RFC 0017
+/// The struct define wrapper for (unsigned 64-bit integer) tx field since
+///
+/// See [tx-since](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0017-tx-valid-since/0017-tx-valid-since.md)
 #[derive(Copy, Clone, Debug)]
 pub struct Since(pub u64);
 
 impl Since {
-    /// TODO(doc): @zhangsoledad
+    /// Wether since represented absolute form
     pub fn is_absolute(self) -> bool {
         self.0 & LOCK_TYPE_FLAG == 0
     }
 
-    /// TODO(doc): @zhangsoledad
+    /// Wether since represented relative form
     #[inline]
     pub fn is_relative(self) -> bool {
         !self.is_absolute()
     }
 
-    /// TODO(doc): @zhangsoledad
+    /// Wether since flag is valid
```

Comment:
```suggestion
    /// Whether since flag is valid
```
Wether is a castrated male sheep. :stuck_out_tongue_closed_eyes:
There are more than one `Wether`.